pkgs/development/python-modules/dataproperty/default.nix

```diff
@@ -0,0 +1,40 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+
+, mbstrdecoder
+, typepy
+
+, pytestCheckHook
+, termcolor
+}:
+
+buildPythonPackage rec {
+  pname = "dataproperty";
+  version = "0.55.0";
+
+  propagatedBuildInputs = [ mbstrdecoder typepy ];
+
+  checkInputs = [ pytestCheckHook termcolor ];
+
+  # Tests fail, even on non-nixos
+  pytestFlagsArray = builtins.concatMap (i: [ "--deselect" i ]) [
```

Comment:
Writing deselect three times won't hurt. Keeping the code clean is more important.